engine/src/damage/core_vector.h

```diff
@@ -0,0 +1,11 @@
+#ifndef COREVECTOR_H
+#define COREVECTOR_H
+
+struct CoreVector {
```

Comment:
I think the Google Code Style would say that all these new structs, classes, etc. should go in one or more namespaces. E.g., `namespace VSDamage` or `namespace VegaStrikeDamage`

Reply:
I'd vote for VegaStrikeDamage.